examples/lang/calls/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "calls"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+ink_core = { path = "../../../core" }
+ink_model = { path = "../../../model" }
+ink_lang = { path = "../../../lang" }
+parity-codec = { version = "4.1.1", default-features = false, features = ["derive"] }
+
+[lib]
+name = "calls"
+crate-type = ["cdylib"]
+
+[features]
```

Comment:
Needs to be adjusted to the new `std` default crate feature compatibility.

Reply:
Still needs adjustment

Reply:
Moved: see https://github.com/paritytech/ink-types-node-runtime/pull/3/files#diff-6f7c540321faf2e34153128ae83caadeR19